k8s.gcr.io/images/k8s-staging-gmsa-webhook/images.yaml

```diff
@@ -0,0 +1,2 @@
+- name: gmsa-webhook
+  dmap:
```

Comment:
```suggestion
# No images yet.
```

Reply:
I have made the tweak, thanks.

Reply:
@phillipsj Sorry for not be clear. What I meant is the file should be empty is there is no images to be promoted. The comment is here to ensure existence of the file. e.g. https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-kind/images.yaml.

Reply:
@ameukam I have updated the PR.